addons/mrp/static/src/scss/mrp_workorder_kanban.scss

```
@@ -9,8 +9,8 @@
                 flex-basis: 100%
             }
         }
-        .o_kanban_record_top {
-            justify-content: space-between;
+        .o_kanban_group .o_kanban_record_top {
+            flex-direction: column;
```

Hello !
I think we should keep these rules for ungrouped kanban records, so adding the `flex-direction` only for the grouped kanban record in addition with what already exist should be better
```suggestion
        .o_kanban_group .o_kanban_record_top {
            flex-direction: column;
        }
        .o_kanban_record_top {
            justify-content: space-between;
```
Because otherwise, we lose the `space-between` for non-grouped record and the tag isn't placed at the same place then before (just after the text instead of the right corner)